mylist.rb

```diff
@@ -0,0 +1,25 @@
+$LOAD_PATH << '.'
+
+require 'myenumerable'
+
+class MyList
```

Comment:
- [ ] Dear @jorgegoco, please use `require_relative` to import `MyEnumerable` since using `require_relative` for files in your directory instead of `require` is a ruby convention. "`Require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory."

Reply:
Ok, thanks.